src/app/base/services/record/record-apollo.service.ts

```diff
@@ -533,13 +534,23 @@ export class RecordApolloService {
   }
 
   //private
-  #tokenMapper(token) {
+  #tokenMapper(token: any, tokensLength: number) {
+    let countLineBreaks = countOcc(token.value, "\n");
+    if (countLineBreaks > 0) {
+      // If we are on the first or last token and either/both of them is new lines, the class w-full cannot work because we don't have a previous or next token, that's why we need the original countLineBreaks
+      // If we are not on the first or last token, the array of the countLineBreaks has to be one less than the actual countLineBreaks because we use the class w-full of the current line as one line break
+      // Adding a completely new line and having a text that needs a new line are different in terms of css classes
+      const checkIfOrLastIdx = token.idx == 0 || token.idx == tokensLength;
```

Comment:
might be wrong but isn't token length to much?
idx starts at 0 so if I understand it correctly it should never be == tokenLength but == tokenLength -1

Reply:
yes that is true, i added this as a lastIdx param so we are not confused and extract the minus -1 when calling the func

Reply:
alright :D